### PR TITLE
Create Adding-Snippets-In-Sublime-Text.md

### PR DESCRIPTION
**Pull request by NormanBenbrahim**
Create a new wiki article about adding snippets to any sublime text installation

### CODE REVIEW

File: Adding-Snippets-In-Sublime-Text.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Adding Snippets To Your Sublime Text Installation
+
+If you are a user of sublime text (I hope you are, it kicks ass. You can get it [here](https://www.sublimetext.com/)), then you may have
+already noticed some of the tab completion capabilities. For example, when you are opening anhtml tag such as `<p>`, it automatically closes
```
Comment (Rafase282) [inline Adding-Snippets-In-Sublime-Text.md]: an html

Comment (Rafase282) [inline Adding-Snippets-In-Sublime-Text.md]: There is a typo where it says "when you are opening anhtml tag such as" it should read "when you are opening a html tag such as"

**Comment by Rafase282** [comment]
@NormanBenbrahim good, please try to [Squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) the commits into one.

**Comment by NormanBenbrahim** [comment]
Sorry I'm still new to this and I did this on my web browser. How would I squash the commits from a browser?

**Comment by Rafase282** [comment]
I don't think it can be done from the browser. You could save the final drag and then redo the PR. Either that or I can check later to see if I can do it for you, I would have to check on that.

**Comment by NormanBenbrahim** [comment]
I'll do that, I'll need to learn how to it anyway. Should I delete this pull request all together?

**Comment by raisedadead** [comment]
@NormanBenbrahim Nope you need not delete the PR. 

Follow these steps to make a copy locally and edit:

1. Clone your fork of the repo on your machine. <br>
`git clone https://github.com/NormanBenbrahim/wiki.git`

2. You will be on the master branch by default. Checkout **a new branch** with same name as you used for this PR i.e `patch-1` <br>
`git checkout -b patch-1`

3. Now you will need to fetch and changes from github. <br>
`git pull patch-1`

4. Check the logs, for listing out all your commits.
`git log --oneline --author=Norman`

5. In addition to the commits already for this PR (2 right now) you will also see 1 commit that that's a merge which we just did withe the git pull in step 3.

6. Now follow proceed with [Squashing](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) .

7. This will update this PR, automatically.

Let us know if you face any issues in the [wiki chat room](https://gitter.im/FreeCodeCamp/Wiki).

**Pull request closed without merging**